examples/cryptocurrency/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 //! for use in production.
 //!
 //! [exonum]: https://github.com/exonum/exonum
-//! [docs]: https://exonum.com/doc/get-started/create-service
+//! [docs]: https://exonum.com/doc/version/latest/get-started/create-service
```

Comment:
In Javadocs we decided to have _versioned_ links so that the API users are directed to the correct documentation version. That helps to prevent a situation where I am on 0.10, click on a link, get to docs for 0.12 (latest) and don't notice, get confused.

Such decision requires updating the links each release, so it is good to have in a release checklist (e.g., https://wiki.bf.local/display/EJB/Java+Binding+Release+Checklist+Template).

Reply:
It is definitely out of scope of this PR, but is something to consider (@Maria-Nosyk , @aleksuss ).

Reply:
Sure, I know why we are doing this in EJB, but I didn't know what's the Exonum policy about this. They seem to not update links too often (see `/storage/#merklized-indices` to `/storage/#merkelized-indices`) so if they wouldn't update this, then IMO it's better to have `latest` instead of fixed version.

Reply:
But sure, if they are ready to update links on every release - then it's better to have a fixed version.

Reply:
I am not sure that updating all the links every release is appropriate for all the docs. We should define clear rules which references require clear version indication and which are just to be kept up-to-date, so that indication of the "latest version" in the link is enough. I put this on my todo list, however, right now it is just a matter of provding readers with the actual information. The details will be dealt with later on.

Reply:
Since people use API docs when working with code of a _particular_ version

> That helps to prevent a situation where I am on 0.10, click on a link, get to docs for 0.12 (latest) and don't notice, get confused.

that is quite important to get to documentation corresponding to _your_ version of Exonum.